src/core/metrics.rs

```diff
@@ -90,6 +124,22 @@ impl BobClient {
     pub(crate) fn get_error_count(&self) {
         self.get_error_count.count(1);
     }
+
+    pub(crate) fn exist_count(&self) {
```

Comment:
Maybe better to use name `exist_counter_inc()` or `exist_inc()`

Reply:
I've tried to save the name policy of other methods. So should I change only this one or all of them?